lte/gateway/c/session_manager/LocalSessionManagerHandler.cpp

```diff
@@ -589,9 +589,9 @@ void LocalSessionManagerHandlerImpl::UpdateTunnelIds(
   auto imsi         = request->sid().id();
   PrintGrpcMessage(static_cast<const google::protobuf::Message&>(request_cpy));
   MLOG(MDEBUG) << "Received a UpdateTunnelIds request for " << imsi
-               << " with default bearer id: " << request->bearer_id()
-               << " enb_teid= " << request->enb_teid()
-               << " agw_teid= " << request->agw_teid();
+               << " with default bearer id:" << request->bearer_id()
+               << ", enb_teid:" << request->enb_teid()
+               << ", agw_teid:" << request->agw_teid();
```

Comment:
you don't like spaces after `:`?

Reply:
nah, let's do spaces. I was just trying to make the =/: changes not space.